site/guide/configuration/customize-your-dashboard.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Customize your dashboard"
+date: last-modified
+---
+
+Personalize what you see on your ValidMind dashboard when you first log in.  Rearrange the position of widgets, and add your own widgets from reporting for quick access.
+
+:::{.callout}
+Changes are automatically saved to your account and will not affect other users.
+:::
+
+
+## Rearrange widgets
+
+1. {{< var link.login >}}
+
+2. On the Dashboard, hover over the title of a widget until the **{{< fa hand>}}** appears. 
+
+3. Click and drag to position your widget as desired.
+
+## Add report widgets
+
+Along with the default **Model Findings** and **Recent Activity** widgets, widgets from [Reports](/guide/model-validation/view-reports.qmd) can be added to your dashboard:
```

Comment:
```suggestion
Along with the default Model Findings and Recent Activity widgets, widgets from [Reports](/guide/model-validation/view-reports.qmd) can be added to your dashboard:
```

Not sure why these default widgets would be bolded here, since the steps don't ask you to interact with them directly. (If I recall our guidelines correctly. 😁  )